src/map/status.cpp

```diff
@@ -2406,6 +2586,10 @@ int status_base_amotion_pc(struct map_session_data* sd, struct status_data* stat
 		val += 1 + skill_lv;
 	if ((skill_lv = pc_checkskill(sd,GS_SINGLEACTION)) > 0 && (sd->status.weapon >= W_REVOLVER && sd->status.weapon <= W_GRENADE))
 		val += ((skill_lv + 1) / 2);
+#ifdef RENEWAL
```

Comment:
We may need to remove this `#ifdef RENEWAL` because we have set `#ifdef RENEWAL_ASPD` above.